ui-modules/app-inspector/app/components/providers/entity-api.provider.js

```diff
@@ -79,8 +79,9 @@ function EntityApi($http, $q) {
         return $http.get('/v1/applications/' + applicationId + '/entities/' + entityId + '/config', {observable: true, ignoreLoadingBar: true});
     }
 
-    function getEntityConfigState(applicationId, entityId, skipResolution) {
-        return $http.get('/v1/applications/' + applicationId + '/entities/' + entityId + `/config/current-state?suppressSecrets=true&skipResolution=${!!skipResolution}`, {observable: true, ignoreLoadingBar: true});
+    function getEntityConfigState(applicationId, entityId, options) {
```

Comment:
Can be cleaner by using string templates, native object deconstruction+extension and default arg value.

```
function getEntityConfigState(applicationId, entityId, options={}) {
    return $http.get(
        `/v1/applications/${applicationId}/entities/${entityId}/config/current-state`,
        { ...options, observable: true, ignoreLoadingBar: true }
    );
}
```